src/analysis/special_functions/integrals.lean

```diff
@@ -35,8 +34,73 @@ by simp only [mul_comm, integral_const_mul]
 lemma integral_div (c : ℝ) : ∫ x in a..b, f x / c = (∫ x in a..b, f x) / c :=
 integral_mul_const c⁻¹
 
+@[simp]
+lemma interval_integrable_pow (n : ℕ) : interval_integrable (λ x, x^n) volume a b :=
+(continuous_pow n).interval_integrable a b
+
+@[simp]
+lemma interval_integrable_id : interval_integrable (λ x, x) volume a b :=
+continuous_id.interval_integrable a b
+
+@[simp]
+lemma interval_integrable_const (c : ℝ) : interval_integrable (λ x, c) volume a b :=
+continuous_const.interval_integrable a b
+
+@[simp]
+lemma interval_integrable.const_mul (c : ℝ) (h : interval_integrable f volume a b) :
```

Comment:
When you have lemmas, like this one, which are valid for all measures and not only for volume, I think it is worth stating them in the more general situation.